src/Playroom/Frame.tsx

```diff
@@ -1,14 +1,12 @@
-import type React from 'react';
-import type { ReactNode } from 'react';
+import { ComponentType, ReactNode } from 'react';
```

Comment:
These should still be `type` imports right? I thought we had a lint rule for that...

Reply:
yes good point, not sure why I removed the `type` import

Reply:
@askoufis the lint rule is there, I think it was added in other PR that moved some things to TypeScript and I forgot to fix that when I rebased!